app/jobs/itms/sync_intensity_job.rb

```diff
@@ -36,4 +41,8 @@ def find_or_create_intensity_source(json, current_source, downloader)
 
     is
   end
+
+  def parse_id(href)
```

Comment:
toto by nebolo lepsie definovat v `ItmsJob`? ak sa predpoklada pouzitie aj pri dalsich entitach

Reply:
necheckoval som ci je to univerzalne funkcne na vsetky entity. ale tiez ma to napadlo tak to checknem a vytiahnem to tam